### DISCUSSION TITLE
Show current water consumption?

### DISCUSSION THREAD

**Discussion by TommySharpNZ** [sebr/bhyve-home-assistant#121] [General]
From what I can tell the attributes of the sensor.zone_1_zone_history entity will show me some stats like total water consumption for the last watering event.

Is it possible to show the current and daily total consumption somehow maybe as it's own entity?

**Comment by sebr** [comment]
Hi @TommySharpNZ this is correct. You should be able to use the [`utility-meter`](https://www.home-assistant.io/integrations/utility_meter/) integration to track over different time periods :)

**Reply by sebr** [reply]
FWIW, I have personally found the water consumption values to be... unreliable? YMMV - good luck!

**Reply by TommySharpNZ** [reply]
Thanks for the reply!
Is there some way to show it in an Entity card like here?
![image](https://user-images.githubusercontent.com/15130284/166228228-507894ec-57ca-418b-be04-be02df89a200.png)

**Reply by sebr** [reply]
Yes you can use the entities created by the utility sensor in the entities card
